ecs-agent/netlib/platform/common_linux_test.go

```diff
@@ -170,6 +170,77 @@ func TestCommon_CreateDNSFiles(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func TestCommon_CreateDNSFilesForDebug(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	for _, testCase := range []struct {
+		name           string
+		resolvConfPath string
+	}{
+		{name: "al", resolvConfPath: "/etc/resolv.conf"},
```

Comment:
should this be just `/etc`? I can see we are appending `/resolv.conf` to resolvConfPath in the test below

Reply:
Thanks for catching that.